src/main/java/com/redhat/devtools/intellij/commonuitest/fixtures/dialogs/FlatWelcomeFrame.java

```diff
@@ -46,11 +53,18 @@
 @FixtureName(name = "Welcome To IntelliJ IDEA Dialog")
 public class FlatWelcomeFrame extends CommonContainerFixture {
     private static final Logger LOGGER = Logger.getLogger(FlatWelcomeFrame.class.getName());
-    private UITestRunner.IdeaVersion intelliJVersion;
+    private static final String JBLIST_XPATH = "//div[@class='JBList']";
```

Comment:
Isn't there some static class holding all labels and stuff? I recall seeing this constant elsewhere as well, in tests maybe?

Reply:
Yes there is the **ButtonLabels** class. Purpose of that class is to keep labels on one place for quick edit if some button labels changes. This is little bit different, maybe creating a class **in the test part of the library** could simplify some test implementation. But as I have written - **in the test part of the library** - that mean outside of the library classes that are being used from outside the library (= just helper class for tests inside library, I do not think people will use them when working with the library+their plugin project). **Should I create that class?**

Reply:
If there are could be a case where you are using some values multiple times, I would go the way to create another class for holding such values. You have one for buttons, nice, lets have for dialog, windows, etc. as well. People do not need to use that, but the framework can. You can also have a final class for similar purposes if shared among implementation. Ie. XPathDefinition which would hold all the xpath expressions to non changing objects in the program. This could be, of course a bigger task to somehow unite this approach. It does not need to be done here, but rather as follow up task ;).

Reply:
Nice idea. As you mentioned, it will probably be bigger task, so I have created an issue for that (https://github.com/redhat-developer/intellij-common-ui-test-library/issues/112) and it will be resolved in another PR. For this PR I will not make any more changes.